src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
-'use strict';
+const tree = document.querySelector('.tree');
 
-// write code here
+tree.addEventListener('click', (branch) => {
+  if (branch.target.tagName === 'LI') {
+    const subTree = branch.target.querySelector('ul');
+
+    if (subTree !== null) {
+      if (subTree.style.display === '' || subTree.style.display === 'block') {
```

Comment:
The condition `subTree.style.display === '' || subTree.style.display === 'block'` is used to check if the subtree is currently visible. However, the initial state when the subtree is not explicitly set might not be an empty string. Consider using `subTree.style.display !== 'none'` to check if the subtree is visible, which covers all cases where the subtree is displayed.